*** Starting uWSGI 2.0.18 (64bit) on [Thu Apr 18 07:48:09 2019] ***
compiled with version: 5.4.0 20160609 on 17 April 2019 23:42:14
os: Linux-4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016
nodename: ubuntu
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/python/django/django/qinyu
writing pidfile to uwsgi.pid
detected binary path: /home/python/.virtualenvs/django/bin/uwsgi
chdir() to /home/python/django/django/qinyu
your processes number limit is 15581
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8080 fd 3
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
Set PythonHome to /home/python/.virtualenvs/django
Python main interpreter initialized at 0x1e16500
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x1e16500 pid: 19908 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 19908)
spawned uWSGI worker 1 (pid: 19910, cores: 2)
spawned uWSGI worker 2 (pid: 19911, cores: 2)
spawned uWSGI worker 3 (pid: 19913, cores: 2)
spawned uWSGI worker 4 (pid: 19914, cores: 2)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
*** Starting uWSGI 2.0.18 (64bit) on [Thu Apr 18 07:49:57 2019] ***
compiled with version: 5.4.0 20160609 on 17 April 2019 23:42:14
os: Linux-4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016
nodename: ubuntu
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/python/django/django/qinyu
writing pidfile to uwsgi.pid
detected binary path: /home/python/.virtualenvs/django/bin/uwsgi
chdir() to /home/python/django/django/qinyu
your processes number limit is 15581
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
probably another instance of uWSGI is running on the same address (127.0.0.1:8080).
bind(): Address already in use [core/socket.c line 769]
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
*** Starting uWSGI 2.0.18 (64bit) on [Thu Apr 18 07:52:27 2019] ***
compiled with version: 5.4.0 20160609 on 17 April 2019 23:42:14
os: Linux-4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016
nodename: ubuntu
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/python/django/django/qinyu
writing pidfile to uwsgi.pid
detected binary path: /home/python/.virtualenvs/django/bin/uwsgi
chdir() to /home/python/django/django/qinyu
your processes number limit is 15581
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8080 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:39838 (port auto-assigned) fd 3
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
Set PythonHome to /home/python/.virtualenvs/django
Python main interpreter initialized at 0x1849600
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x1849600 pid: 20020 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 20020)
spawned uWSGI worker 1 (pid: 20022, cores: 2)
spawned uWSGI worker 2 (pid: 20023, cores: 2)
spawned uWSGI worker 3 (pid: 20024, cores: 2)
spawned uWSGI worker 4 (pid: 20025, cores: 2)
spawned uWSGI http 1 (pid: 20029)
[pid: 20022|app: 0|req: 1/1] 127.0.0.1 () {38 vars in 751 bytes} [Thu Apr 18 07:52:31 2019] GET /goods/index => generated 6266 bytes in 1828 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
Not Found: /static/css/reset.css
[pid: 20022|app: 0|req: 2/2] 127.0.0.1 () {38 vars in 738 bytes} [Thu Apr 18 07:52:33 2019] GET /static/css/reset.css => generated 2641 bytes in 22 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/css/main.css
[pid: 20022|app: 0|req: 3/3] 127.0.0.1 () {38 vars in 736 bytes} [Thu Apr 18 07:52:33 2019] GET /static/css/main.css => generated 2638 bytes in 13 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/js/jquery-1.12.4.min.js
[pid: 20022|app: 0|req: 4/4] 127.0.0.1 () {38 vars in 743 bytes} [Thu Apr 18 07:52:33 2019] GET /static/js/jquery-1.12.4.min.js => generated 2671 bytes in 10 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/js/jquery-ui.min.js
[pid: 20022|app: 0|req: 5/5] 127.0.0.1 () {38 vars in 735 bytes} [Thu Apr 18 07:52:33 2019] GET /static/js/jquery-ui.min.js => generated 2659 bytes in 34 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/js/slide.js
Not Found: /static/images/logo.png
[pid: 20022|app: 0|req: 6/7] 127.0.0.1 () {38 vars in 738 bytes} [Thu Apr 18 07:52:34 2019] GET /static/images/logo.png => generated 2647 bytes in 22 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
[pid: 20024|app: 0|req: 1/7] 127.0.0.1 () {38 vars in 719 bytes} [Thu Apr 18 07:52:33 2019] GET /static/js/slide.js => generated 2635 bytes in 1084 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/js/jquery-1.12.4.min.js
[pid: 20022|app: 0|req: 7/8] 127.0.0.1 () {40 vars in 775 bytes} [Thu Apr 18 07:52:34 2019] GET /static/js/jquery-1.12.4.min.js => generated 2671 bytes in 10 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 20022|app: 0|req: 8/9] 127.0.0.1 () {36 vars in 668 bytes} [Thu Apr 18 07:52:34 2019] GET /favicon.ico => generated 2614 bytes in 9 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Gracefully killing worker 1 (pid: 20022)...
gateway "uWSGI http 1" has been buried (pid: 20029)
...brutally killing workers...
worker 2 buried after 13 seconds
worker 4 buried after 13 seconds
worker 1 buried after 14 seconds
worker 3 buried after 14 seconds
binary reloading uWSGI...
chdir() to /home/python/django/django/qinyu
closing all non-uwsgi socket fds > 2 (max_fd = 1048576)...
found fd 3 mapped to socket 0 (127.0.0.1:39838)
running /home/python/.virtualenvs/django/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.18 (64bit) on [Thu Apr 18 11:27:03 2019] ***
compiled with version: 5.4.0 20160609 on 17 April 2019 23:42:14
os: Linux-4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016
nodename: ubuntu
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/python/django/django/qinyu
detected binary path: /home/python/.virtualenvs/django/bin/uwsgi
chdir() to /home/python/django/django/qinyu
your processes number limit is 15581
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8080 fd 5
uwsgi socket 0 bound to TCP address 127.0.0.1:34332 (port auto-assigned) fd 4
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
Set PythonHome to /home/python/.virtualenvs/django
Python main interpreter initialized at 0x23693b0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x23693b0 pid: 20020 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 20020)
spawned uWSGI worker 1 (pid: 22242, cores: 2)
spawned uWSGI worker 2 (pid: 22243, cores: 2)
spawned uWSGI worker 3 (pid: 22246, cores: 2)
spawned uWSGI worker 4 (pid: 22247, cores: 2)
spawned uWSGI http 1 (pid: 22250)
*** Starting uWSGI 2.0.18 (64bit) on [Thu Apr 18 15:21:45 2019] ***
compiled with version: 5.4.0 20160609 on 17 April 2019 23:42:14
os: Linux-4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016
nodename: ubuntu
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/python/django/django/qinyu
writing pidfile to uwsgi.pid
detected binary path: /home/python/.virtualenvs/django/bin/uwsgi
chdir() to /home/python/django/django/qinyu
your processes number limit is 15581
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8080 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:40863 (port auto-assigned) fd 3
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
Set PythonHome to /home/python/.virtualenvs/django
Python main interpreter initialized at 0xcca4f0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0xcca4f0 pid: 4895 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 4895)
spawned uWSGI worker 1 (pid: 4897, cores: 2)
spawned uWSGI worker 2 (pid: 4898, cores: 2)
spawned uWSGI worker 3 (pid: 4899, cores: 2)
spawned uWSGI worker 4 (pid: 4900, cores: 2)
spawned uWSGI http 1 (pid: 4901)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 4901)
worker 1 buried after 5 seconds
worker 2 buried after 5 seconds
worker 3 buried after 5 seconds
worker 4 buried after 5 seconds
goodbye to uWSGI.
